src/cr/cube/cube.py

```diff
@@ -455,6 +455,11 @@ def _cube_response(self):
                 "(str) or dict." % type(self._cube_response_arg).__name__
             )
 
+    @lazyproperty
+    def _has_missing(self):
```

Comment:
Now that I read it, it doesn't really have to do anything with "has missing", because it doesn't even check the measures. We should figure out a better name for this. I now why you wanted to use "show", but it's still wrong. Let me think about it.